facilitator/src/sample.rs

```diff
@@ -128,16 +123,21 @@ pub fn generate_ingestion_sample(
 
                         let (pha_share, facilitator_share) = client
                             .encode_simple(&data)
-                            .context("failed to encode data")?;
+                            .map_err(|e| anyhow!("failed to encode data: {:?}", e))?;
```

Comment:
Was this necessary? `.context()` should preserve the underlying error, unless I have missed something.